ethcore/src/engines/validator_set/simple_list.rs

```diff
@@ -33,9 +33,14 @@ pub struct SimpleList {
 impl SimpleList {
 	/// Create a new `SimpleList`.
 	pub fn new(validators: Vec<Address>) -> Self {
-		SimpleList {
-			validators: validators,
+		let validator_count = validators.len();
+		if validator_count == 1 {
+			warn!(target: "engine", "Running AuRa with a single validator is not supported.");
```

Comment:
In principle it's fine, but we should emphasize that in such setup we have instant finality and it might be better to use some database instead of a blockchain.